Runtime/Behaviours/CinemachineBrain.cs

```diff
@@ -680,8 +691,8 @@ private void ProcessActiveCamera(float deltaTime)
                 // No active virtal camera.  We create a state representing its position
                 // and call the callback, but we don't actively set the transform or lens
                 var state = CameraState.Default;
-                state.RawPosition = transform.position;
-                state.RawOrientation = transform.rotation;
+                state.RawPosition = TargetOverride.transform.position;
```

Comment:
Performance: better to call TargetOverride just once in this method, into a local variable, and use that.  Otherwise there is a bunch of code run for nothing